File: app/src/main/java/com/majateam/bikespot/Global.java
package com.majateam.bikespot;

/**
 * Created by Nicolas Martino on 10/06/15.
 */
public class Global {

    private static final Boolean STAGING = false;
    public static final String ENDPOINT = "http://api.velocalisateur.com/";
    private static final String STAGING_URL = "staging/";
    public static final String BIKES_URL = ((STAGING) ? STAGING_URL : "") + "bikes";
    public static final String DOCKS_URL = ((STAGING) ? STAGING_URL : "") + "docks";
}

File: app/src/main/java/com/majateam/bikespot/Global.kt
package com.majateam.bikespot

/**
 * Created by Nicolas Martino on 10/06/15.
 */
object Global {
    private const val STAGING = false
    const val ENDPOINT = "http://api.velocalisateur.com/"
    private const val STAGING_URL = "staging/"
    val BIKES_URL = (if (STAGING) STAGING_URL else "") + "bikes"
    val DOCKS_URL = (if (STAGING) STAGING_URL else "") + "docks"
}